Formula/wuzz.rb

```diff
@@ -0,0 +1,16 @@
+class Wuzz < Formula
+  desc "Interactive cli tool for HTTP inspection"
+  homepage "https://github.com/asciimoo/wuzz"
+  url "https://github.com/asciimoo/wuzz/releases/download/v0.4.0/wuzz_darwin_amd64"
```

Comment:
Maybe `v#{version}` instead of `v0.4.0`

Reply:
It would be nice, but looks like it's not possible: the coding standard requires that an url should be defined before version. And code, of course, substitutes variable with an empty value if I set version after url.